workflow-job

```diff
@@ -466,6 +476,26 @@ flows["${sha1}"] = {
             dockeringGuiceCassandraLdap.success()
         }
 
+        stage "Deploying Guice Server with JPA then provision data ${buildId}"
+        node {
+            sh "cd dockerfiles/run/guice/provisioned; cp Dockerfile NewDockerfile; sed -i -- 's,linagora/james-jpa-guice,${images.jamesJPA},g' NewDockerfile"
```

Comment:
The CI should clone project from github on specified branch for testing but it does not commit any changes. But yes, to avoid build new image on that folder and publish it later. It will be deleted for safely.